<launch>
  <arg name="data_folder" default="/home/x-arm/macaca/data/handeye/eyetracker"/>
  <arg name="rate" default="30"/>
  <node pkg="macaca_tf_setup" type="macaca_tf_setup_node" name="macaca_tf_setup" output="screen">
    <param name="lscene_tracker0_tf_datafile" type="str" value="$(arg data_folder)/lscene_tracker0.yml"/>
    <param name="robot_tracker1_tf_datafile" type="str" value="$(arg data_folder)/robot_tracker1.yml"/>
    <param name="joint6_tracker2_tf_datafile" type="str" value="$(arg data_folder)/joint6_tracker2.yml"/>
    <param name="rate" type="int" value="$(arg rate)"/>
  </node>
</launch>
